jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/AllowedResourceAliasChecker.java

```diff
@@ -192,6 +206,26 @@ protected boolean isAllowed(Path path)
         return false;
     }
 
+    protected boolean isSameFile(Path path, Resource resource)
```

Comment:
We could have `isSameFile(Path)` as a method on `Resource`, then only `CombinedResource` would need the iteration.

Reply:
@lachlan-roberts  bump